{"status": "failed", "name": "Should be able to use properties from match in ON MATCH and ON CREATE", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "skip"}, {"name": "feature", "value": "MergeNodeAcceptance"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython2"}], "stop": 1578486089355, "historyId": "90a53ad98a8d04df16c7c45aa9c295d6", "start": 1578486089130, "steps": [{"status": "passed", "start": 1578486089133, "stop": 1578486089344, "name": "Given an empty graph"}, {"status": "passed", "start": 1578486089345, "stop": 1578486089348, "name": "And having executed", "attachments": [{"source": "a17d0c19-95c1-4bd7-a55e-a24c358208ff-attachment.txt", "type": "text/plain", "name": ".text"}]}, {"status": "passed", "start": 1578486089349, "stop": 1578486089352, "name": "When executing query", "attachments": [{"source": "b2c6f125-f4ba-48a7-a2af-0b8934635857-attachment.txt", "type": "text/plain", "name": ".text"}]}, {"status": "failed", "attachments": [{"source": "fc3652e1-117f-487a-bfe7-2708a9fab250-attachment.csv", "type": "text/csv", "name": ".table"}], "stop": 1578486089355, "start": 1578486089353, "statusDetails": {"message": "AssertionError\n", "trace": "  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/Steps.py\", line 86, in step_impl\n    assertions.assert_resultset_length(resultset, expected_length)\n  File \"/home/laptop-linux/dev/redis/graph/tests/tck/features/steps/../../utils/assertions.py\", line 195, in assert_resultset_length\n    assert(len(resultset.result_set) == length)\n"}, "name": "Then the result should be"}, {"status": "skipped", "start": 1578486089355, "stop": 1578486089355, "name": "And the side effects should be", "attachments": [{"source": "87545435-2ab2-444d-8814-44e8b33837cd-attachment.csv", "type": "text/csv", "name": ".table"}]}], "statusDetails": {"message": "AssertionError\n", "trace": "  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/Steps.py\", line 86, in step_impl\n    assertions.assert_resultset_length(resultset, expected_length)\n  File \"/home/laptop-linux/dev/redis/graph/tests/tck/features/steps/../../utils/assertions.py\", line 195, in assert_resultset_length\n    assert(len(resultset.result_set) == length)\n"}, "uuid": "74b56b0b-8d58-4c0f-aaf9-12cb51064649"}